core/tauri/permissions/.gitignore

```diff
@@ -1 +1 @@
-.schema.json
+*/schemas/schema.json
```

Comment:
the schema files are in `permissions/window/.schema.json` and so on, shouldn't this be
```suggestion
.schema.json
```

Reply:
i guess i forgot to delete some